libsolidity/parsing/Parser.cpp

```diff
@@ -962,7 +965,26 @@ ASTPointer<UsingForDirective> Parser::parseUsingDirective()
 	ASTNodeFactory nodeFactory(*this);
 
 	expectToken(Token::Using);
-	ASTPointer<IdentifierPath> library(parseIdentifierPath());
+
+	vector<ASTPointer<IdentifierPath>> functions;
+	bool const usesBraces = m_scanner->currentToken() == Token::LBrace;
+	if (usesBraces)
+	{
+		vector<ASTPointer<IdentifierPath>> functionList;
```

Comment:
Not sure this helps much compared to just adding to ``functions`` directly...

Reply:
Oh, I think I even changed that, but it seems the change got lost.